relayer/src/worker/packet.rs

```diff
@@ -11,11 +11,27 @@ use crate::object::Packet;
 use crate::telemetry;
 use crate::util::retry::{retry_with_index, RetryResult};
 use crate::util::task::{spawn_background_task, Next, TaskError, TaskHandle};
-use crate::worker::retry_strategy;
 
 use super::error::RunError;
 use super::WorkerCmd;
 
+mod retry_strategy {
+    use core::time::Duration;
+
+    use retry::delay::Fibonacci;
+
+    use crate::util::retry::clamp_total;
+
+    // Default parameters for the retrying mechanism
+    const MAX_DELAY: Duration = Duration::from_secs(2); // 1 minute
+    const MAX_TOTAL_DELAY: Duration = Duration::from_secs(5); // 10 minutes
+    const INITIAL_DELAY: Duration = Duration::from_secs(1); // 1 second
+
+    pub fn default() -> impl Iterator<Item = Duration> {
```

Comment:
Could we add some documentation for this to briefly explain the logic of the default retry strategy? Putting the logic in a `default` method doesn't give any indication via the naming what the strategy is.

Reply:
Yeah I don't really like the `retry` crate we are currently using. It is overly complicated and do not blend well with the idiomatic use of `Result`. It is worth refactoring to simplify the logic.

For this case, I'm just copying the boilerplate from other code, because the current strategies are not very customizable. We'll figure out a proper strategy in a proper PR. Here we just want to identify the place that needs to be modified for the relayer to fail gracefully.

Reply:
Ok, should probably add an issue or TODO to catalog this 🙂

Reply:
I am reverting these changes as it is unclear if they are necessary. I agree this is the right place to customize the retry strategy in case we need to adapt the packet command worker retrying behavior (but that's not clear to me we need to do). 

8eea79fd